filter_plugins/teku_filters.py

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env python
+
+from typing import Union, List
+
+
+class FilterModule(object):
+    def filters(self):
+        return {
+            'teku_opts_format': self.tekuOptsFormat,
+        }
+
+    def tekuOptsFormat(self, teku_env_opts: Union[str, List[str]]):
+        if isinstance(teku_env_opts, str):
+            teku_env_opts = teku_env_opts.split()
+
+        escaped_teku_env_opts = [item.replace('"', '\\"') for item in teku_env_opts]
+
+        return f'''"{'" "'.join(escaped_teku_env_opts)}"'''
```

Comment:
lolwut?

Reply:
https://www.python.org/dev/peps/pep-0498/

TIL!

Reply:
This is super not readable, sadly... Perhaps just add a comment like `# quote the individual items`?